lib/rentlinx/models/base.rb

```diff
@@ -44,6 +40,10 @@ def get_units_for_property_id(id)
       Rentlinx.client.get_units_for_property_id(id)
     end
 
+    def get_photos_for_property_id(id)
```

Comment:
This seems out of place in the base model class.  Would <code>Property</code> be a better place for it?  (The same line of reasoning applies to <code>get_units_for_property_id</code>, but that's not something you changed).